creator-node/src/components/contentBlacklist/contentBlacklistComponentService.ts

```diff
@@ -3,12 +3,28 @@ const models = require('../../models')
 
 const types = models.ContentBlacklist.Types
 
-const getAllTrackIds = async () => {
+type BlacklistedContent = {
+  trackIds: Promise<string[]>
+  userIds: Promise<number[]>
+  individualSegments: string[]
+  numberOfSegments: number
+  allSegments: string[]
+}
+
+type ICBLSegments = {
```

Comment:
One more nit: Can you change the name of the type to just `Segment`

Reply:
done